app/workers/evss/disability_compensation_form/job_status.rb

```diff
@@ -15,28 +15,26 @@ module JobStatus
         # @param msg [Hash] The message payload from Sidekiq
         #
         def job_exhausted(msg)
-          submission_id = msg['args'].first
-          jid = msg['jid']
-          error_message = msg['error_message']
-          klass = self.class.name.demodulize
```

Comment:
`self.class.name.demodulize` is always `Class` 😞

Reply:
If the above suggestion works (adding .new before .class) it might work the same here.

Reply:
yeah, `msg['class'].demodulize` works too.